aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -37,6 +44,16 @@ private void Awake()
             Instance = this;
         }
 
+        // Get the equipment inventory if it is not set.
+        if (!_equipmentInventory)
+        {
+            _equipmentInventory = _playerTransform.gameObject.GetComponentInChildren<EquipmentInventory>();
+            if (!_equipmentInventory)
+            {
+                Debug.LogError("Equipment Inventory not found.");
+            }
+        }
```

Comment:
Tip I got from Jens: do this in the equipmentinventory class to prevent bloating the inputmanager too much.